packages/react/src/patterns/blocks/index.js

```diff
@@ -18,3 +18,4 @@ export * from './ContentGroupSimple';
 export * from './FeatureCardBlockMedium';
 export * from './FeatureCardBlockLarge';
 export * from './LogoGrid';
+export * from './LeadSpaceBlock';
```

Comment:
Add this alphabetically (should be before `LogoGrid`)

Reply:
changed the order.